src/Spice86.Core/Emulator/Memory/Memory.cs

```diff
@@ -252,6 +252,11 @@ public class Memory {
         get;
     }
 
+    /// <summary>
+    /// Is A20 gate rollover enabled or not
+    /// </summary>
+    public bool IsA20Enabled { get; set; }
```

Comment:
Depending on that, physical addresses should wrap around the ram. Not sure this is something to do in this PR.
With it disabled memory address should wrap with a modulo of 0xFFFFF
With it enabled, the modulo is (0xFFFF*16 + 0xFFFF) which is 0x10FFEF

Reply:
Code removed.